src/scrabble.js

```diff
@@ -12,8 +12,7 @@ class Scrabble {
   }
 
   getCharScore (char) {
-    if (ScoreUtil.scores[char]) return ScoreUtil.scores[char]
-    else return 0
+    return ScoreUtil.scores[char] || 0
```

Comment:
The or 0 was very efficient and line efficient, it even executed one of the test. Simple and clean code is the way 👌